src/remote-control/screenshare-connection.js

```diff
@@ -0,0 +1,127 @@
+import { JitsiMeetJSProvider } from 'vendor';
+
+/**
+ * Manages the ProxyConnectionService between between a remote control and a
+ * participant in a Jitsi-Meet meeting.
+ */
+export default class ScreenshareConnection {
+    /**
+     * Initializes a new {@code ScreenshareConnection} instance.
+     *
+     * @param {Object} options - Configuration to initialize with.
+     * @param {Function} options.sendMessage - Callback invoked when the
+     * {@code ProxyConnectionService} needs to send a message out to Jitsi-Meet.
+     * @param {Function} options.onConnectionClosed - Callback to invoke when a
+     * the connection or part of the connection has closed itself.
+     */
+    constructor(options) {
+        this.options = options;
+
+        /**
+         * Whether or not the {@code ProxyConnectionService} has an active
+         * connection. Used to prevent connection events from firing after the
+         * connection has been stopped.
+         *
+         * @type {boolean}
+         */
+        this._isStopped = true;
```

Comment:
`_isActive`?  the weird thing about `_isStopped` is you kinda get a double negative in the `if (!_isStopped)` case

Reply:
Fair. You are correct; state should be positive to avoid the double negative.